app/assets/stylesheets/cats.css.scss

```diff
@@ -0,0 +1,58 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+
+.container#main-container {
+  margin-top: 80px;
+  margin-right: auto;
+  margin-left: auto;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.navbar.navbar-default.navbar-fixed-top {
+  background-color: rgba(50,50,50,1);
+}
+
+.navbar-inverse .navbar-nav > li > a  {
+  font-family: 'Comfortaa', Arial, serif; font-weight: 400;
+  text-transform: uppercase;
+}
+
+.yield-style {
+  font-family: 'Open Sans Condensed', Arial, serif; font-weight: 400;
+  font-size: 1.5em;
+}
+
+.list {
+  margin-top: 20px;
+  margin-bottom: 60px;
+  text-align: center;
+}
+
+
+h1 {
+  font-family: 'Comfortaa', Arial, serif; font-weight: 400;
+  font-size: 1.5em;
+  text-align: center;
+}
+
+#logo-home > h1 {
+  font-family: 'Comfortaa', Arial, serif; font-weight: 400;
```

Comment:
Properties should be ordered font-family, font-size, font-weight, font-weight<br>Property 'font-weight' should be placed on own line